cms/envs/devstack.py

```diff
@@ -79,8 +80,13 @@
 ################################ DEBUG TOOLBAR ################################
 
 INSTALLED_APPS += ['debug_toolbar']
-
 MIDDLEWARE.append('debug_toolbar.middleware.DebugToolbarMiddleware')
+
+################################ DJANGO SILK ################################
+if DEBUG:
```

Comment:
`DEBUG` flag is set for the Dev server, shouldn't we add `silk` pkg in requirements?